.travis.yml

```diff
@@ -10,6 +10,7 @@ jobs:
   include:
     - script:
       - bin/scalafmt --test
+    - script:
```

Comment:
What's the motivation?

Reply:
In a previous commit, the `bin/scalafmt` command failed without failing the job immediately. Then, the `sbt test...` command succeeded so the Travis job failed but the failure was hidden halfway the output.

This way makes it very clear what the issue really is.

To be honest, I'd event split 

```yaml
  - script:
    - sbt test headerCheck test:headerCheck` 
```

into

```yaml
  - script:
    - sbt headerCheck test:headerCheck` 
  - script:
    - sbt test
```

Reply:
In that case, there's a better solution.

The problem with splitting into multiple jobs is that it causes overhead, so it's best to minimise.

To have this fail fast and obviously I suggest:

```yaml
  - script:
    - bin/scalafmt --test || travis_terminate 1
    - sbt headerCheck test:headerCheck test
```

Reply:
Hmm, but then a silly formatting issue masks other problems in tests or header checks.

I want all the info asap and easily accessible. 

Ok, I'll revert this and we can discuss on a separate PR.